app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -214,6 +214,55 @@ setTimeout(function() {
 // Enables noSearchHidden plugin
 $.fn.dataTable.defaults.noSearchHidden = true;
 
+$('form#form-export').submit(function(e) {
+  var form = this;
+
+  if (currentMode === 'viewMode') {
+    // Remove all hidden fields
+    $('#form-export').find('input[name=row_ids\\[\\]]').remove();
+    $('#form-export').find('input[name=header_ids\\[\\]]').remove();
+
+    // Append samples
+    appendSamplesIdToForm(form);
+
+    // Append visible column information
+    $('.active table#repository-table thead tr').children('th')
+                                                .each(function(i) {
+      var th = $(this);
+      var val;
+      if ($(th).attr('id') === 'checkbox' || $(th).attr('id') === 'assigned')
```

Comment:
Check for 'assigned' is not needed because export is only available on repositories page, not inside of a task, where the records have 'assigned' column.